certbot-nginx/certbot_nginx/constants.py

```diff
@@ -1,9 +1,14 @@
 """nginx plugin constants."""
 import pkg_resources
+import platform
 
-
+if(platform.system() == ('FreeBSD' or 'Darwin')):
+    server_root_tmp = "/usr/local/etc/nginx"
+else:
+    server_root_tmp = "/etc/nginx"
+    
 CLI_DEFAULTS = dict(
-    server_root="/etc/nginx",
+    server_root=server_root_tmp
```

Comment:
tests are failing because it's missing a comma here

Reply:
I tried your command but it failed, 

certbot: error: unrecognized arguments: --server-root /usr/local/etc/nginx

And sorry for comma, it should be like 

server_root=server_root_tmp,

I tested like this and worked.

Reply:
You're right, it should have been `--nginx-server-root`.

Great! Could you add another commit to fix up the PR?